gstools/field/rng.py

```diff
@@ -16,56 +26,62 @@ class RNG(object):
     It only generates isotropic fields, as anisotropic fields can be created by
     coordinate transformations.
 
-    Parameters
-    ----------
-        dim : int
-            spatial dimension
-        seed : int, optional
-            set the seed of the master RNG, if "None",
-            a random seed is used
-
     Examples
     --------
         >>> r = RNG(dim = 2, seed=76386181534)
         >>> Z, k = r(model='gau', len_scale=10., mode_no=100)
     """
     def __init__(self, dim, seed=None):
+        '''Initialize a random number generator
+
+        Parameters
+        ----------
+            dim : :class:`int`
+                spatial dimension
+            seed : :class:`int`, optional
+                set the seed of the master RNG, if "None",
+                a random seed is used
+        '''
         if dim < 1 or dim > 3:
             raise ValueError('Only dimensions of 1 <= d <= 3 are supported.')
         self._dim = dim
+        # set seed
+        self._seed = None
+        self._master_RNG_fct = None
+        self._master_RNG = None
         self.seed = seed
 
         self._trans_1d = lambda len_scale, r: r / len_scale
         self._trans_2d = [lambda len_scale, r, phi:
-                              r*np.cos(phi) / len_scale,
+                          r*np.cos(phi) / len_scale,
                           lambda len_scale, r, phi:
-                              r*np.sin(phi) / len_scale]
+                          r*np.sin(phi) / len_scale]
         self._trans_3d = [lambda len_scale, r, theta, phi:
-                              r * np.sin(theta) * np.cos(phi) / len_scale,
+                          r * np.sin(theta) * np.cos(phi) / len_scale,
                           lambda len_scale, r, theta, phi:
-                              r * np.sin(theta) * np.sin(phi) / len_scale,
+                          r * np.sin(theta) * np.sin(phi) / len_scale,
                           lambda len_scale, r, theta, phi=None:
-                              r * np.cos(theta) / len_scale]
+                          r * np.cos(theta) / len_scale]
```

Comment:
I really think the pep8 standard makes things worse here

Reply:
OK. We can just turn back to the original indentation.